docs/docs/platform/templates.md

```diff
@@ -6,33 +6,33 @@ sidebar_position: 1
 
 A template holds the entire flow of messages sent to the subscriber. This is where all the different channels are tied together under a single entity.
 
-Let's explore the different parts of a notification template
+Let's explore the different parts of a notification template.
 
 ## Global template metadata
 
 This will contain general information regarding the notification itself. Let's explore the different aspects of it.
 
 ### Notification name
 
-The name will be used to identify the notification template when triggering it. A slugified version of the name will be generated after the notification was created. For example, a notification template with the name of "Test Notification" will be converted to "test-notification" as the trigger key.
+The name used to identify the notification template when triggering it. A slugified version of the name will be generated after the notification was created. For example, a notification template with the name of "Test Notification" will be converted to "test-notification" as the trigger key.
 
 ### Notification Group
 
-The notification group is used to group multiple notification templates into a single group, currently only used behind the scenes for organisational purposes. But, in the upcoming subscriber preferences management, it will be used to group multiple notifications for the subscriber.
+The notification group is used to group multiple notification templates into a single group (currently only used behind the scenes for organizational purposes). But, in the upcoming subscriber preferences management, it will be used to group multiple notifications for the subscriber.
 
 ## Template steps
 
-The templates steps are used to organize the different messages in a particular flow. You can add messages for multiple channels and in the upcoming workflow editor release you will be able to configure custom flows that includes actions like: Delay, Digest and other conditional steps.
+The template steps are used to organize the different messages in a particular flow. You can add messages for multiple channels and in the upcoming workflow editor release you will be able to configure custom flows that include actions like Delay, Digest and other conditional steps.
 
 ## Messages
 
 A message is particularly tied to a specific channel and will create the content template associated with its channel.
 
-For email channel, you can either use our basic visual editor or a fully custom code with [handlebars variables](https://handlebarsjs.com/guide/).
+For email channels, you can either use our basic visual editor or a fully custom code with [handlebars variables](https://handlebarsjs.com/guide/).
```

Comment:
Channel in Novu is any type of notification used: email, sms, etc. So the use here was rightfully singular. Please roll back the change.